client/mysql.cc

```diff
@@ -1128,6 +1146,8 @@ int main(int argc,char *argv[])
   MY_INIT(argv[0]);
   DBUG_ENTER("main");
   DBUG_PROCESS(argv[0]);
+
+  redact_mysql_pwd();
```

Comment:
so early in the process effectively disables its use.

Reply:
note: used by scripts/wsrep_sst_mysqldump.sh

Reply:
See my previous comment, a following `getenv("MYSQL_PWD")` still returns the original password.